app/src/main/java/com/cornellappdev/android/eatery/components/BarChartRoundedRenderer.java

```diff
@@ -0,0 +1,183 @@
+package com.cornellappdev.android.eatery.components;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+
+import com.github.mikephil.charting.animation.ChartAnimator;
+import com.github.mikephil.charting.buffer.BarBuffer;
+import com.github.mikephil.charting.data.BarData;
+import com.github.mikephil.charting.data.BarEntry;
+import com.github.mikephil.charting.highlight.Highlight;
+import com.github.mikephil.charting.highlight.Range;
+import com.github.mikephil.charting.interfaces.dataprovider.BarDataProvider;
+import com.github.mikephil.charting.interfaces.datasets.IBarDataSet;
+import com.github.mikephil.charting.renderer.BarChartRenderer;
+import com.github.mikephil.charting.utils.Transformer;
+import com.github.mikephil.charting.utils.Utils;
+import com.github.mikephil.charting.utils.ViewPortHandler;
+import com.github.mikephil.charting.model.GradientColor;
+import android.graphics.LinearGradient;
+
+public class BarChartRoundedRenderer extends BarChartRenderer {
```

Comment:
leave a comment here explaining the class and where to get more info (ie link the repo)